src/Libraries/RevitNodes/Elements/DirectShape.cs

```diff
@@ -220,17 +220,27 @@ private static IList<GeometryObject> GenerateTessellatedGeo(DesignScriptEntity s
             }
             else if (shapeReference is Autodesk.DesignScript.Geometry.Surface)
             {
-                tessellatedShape = (shapeReference as Autodesk.DesignScript.Geometry.Surface).ToRevitType(
-                    TessellatedShapeBuilderTarget.AnyGeometry,
-                    TessellatedShapeBuilderFallback.Mesh,
-                    MaterialId: materialId);
+                var geometry = ProtoToRevitSolid.ToRevitType(shapeReference as Autodesk.DesignScript.Geometry.Surface);
+                if(geometry != null)
+                {
+                    tessellatedShape = new List<GeometryObject>() { geometry };
+                }
+                else
+                {
+                    tessellatedShape = (shapeReference as Autodesk.DesignScript.Geometry.Surface).ToRevitType(TessellatedShapeBuilderTarget.AnyGeometry, TessellatedShapeBuilderFallback.Mesh, MaterialId: materialId);
```

Comment:
can you add a comment to these cases stating that if the BRep is null, we fallback to the tessellated shape builder?